crates/api_models/src/payments.rs

```diff
@@ -711,6 +711,8 @@ pub enum WalletData {
     PaypalSdk(PayPalWalletData),
     /// The wallet data for WeChat Pay Redirection
     WeChatPayRedirect(Box<WeChatPayRedirection>),
+    //The wallet data for Cashapp
+    Cashapp(Box<CashAppRedirection>),
```

Comment:
Nit: Please follow consistent naming, one of `Cashapp` or `CashApp` throughout the codebase.